### PR TITLE
Update system df docs to call out performance implications

### PR DESCRIPTION
**Pull request by wsong**
Signed-off-by: Wayne Song <wsong@docker.com>

### CODE REVIEW

File: docs/reference/commandline/system_df.md [hunk 1]
```diff
@@ -77,6 +77,15 @@ my-named-vol                                                       0
 > **Note**: Network information is not shown because it doesn't consume the disk
 > space.
 
+## Performance
+
+The system df command can be very resource-intensive. It traverses the
```
Comment (thaJeztah) [inline docs/reference/commandline/system_df.md]: can you add backticks around `system df` ?

**Review by thaJeztah — CHANGES_REQUESTED** [review]
one minor nit, but LGTM otherwise, thanks!

File: docs/reference/commandline/system_df.md (hunk 1 above)
Comment (wsong) [inline docs/reference/commandline/system_df.md]: Done.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Comment by thaJeztah** [comment]
ping @mlaventure @vdemeester PTAL

File: docs/reference/commandline/system_df.md [hunk 2]
```diff
@@ -77,6 +77,15 @@ my-named-vol                                                       0
 > **Note**: Network information is not shown because it doesn't consume the disk
 > space.
 
+## Performance
+
+The `system df` command can be very resource-intensive. It traverses the
+filesystem of every container and every volume in the system. You should be
```
Comment (mlaventure) [inline docs/reference/commandline/system_df.md]: `every image, container, and volume`

File: docs/reference/commandline/system_df.md [hunk 3]
```diff
@@ -77,6 +77,15 @@ my-named-vol                                                       0
...
+careful running this command in systems with lots of containers or volumes or
+in systems where some containers or volumes have very large filesystems with
```
Comment (mlaventure) [inline docs/reference/commandline/system_df.md]: `where some images, containers, or volumes`

**Review by mlaventure — CHANGES_REQUESTED** [review]

**Comment by wsong** [comment]
Addressed comments.

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

**Pull request merged**